pkg/cmd/cli/cmd/export.go

```diff
@@ -92,9 +94,16 @@ func RunExport(f *clientcmd.Factory, exporter Exporter, in io.Reader, out io.Wri
 	if err != nil {
 		return err
 	}
-	outputVersion, err := kcmdutil.OutputVersion(cmd, clientConfig.GroupVersion)
-	if err != nil {
-		return err
+
+	var outputVersion schema.GroupVersion
+	outputVersionString := kcmdutil.GetFlagString(cmd, "output-version")
+	if len(outputVersionString) == 0 {
+		outputVersion = *clientConfig.GroupVersion
```

Comment:
Isn't this going to be incorrect every time that you're getting resources from multiple groups.  `oc export builds,deployments`.  I really think we want to move away from choosing the `--output-version` like upstream and using use `oc convert` instead.  We can make that one spot correct with a preferred version chain.

I have no suggestion that I think could make this work properly for the scenario I described.